frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/GitMacheteGraphTable.java

```diff
@@ -129,8 +129,8 @@ public void setBranchLayout(IBranchLayout newBranchLayout) {
     this.branchLayout = newBranchLayout;
   }
 
-  public void setMacheteFilePath(Path newMacheteFilePath) {
-    this.macheteFilePath = newMacheteFilePath;
+  public void setBranchLayoutManager(IBranchLayoutManager newBranchLayoutManager) {
```

Comment:
`@Setter` for this one and other fields? unless there are still some problems with Lombok/Checker